MapboxCoreNavigation.podspec

```diff
@@ -40,7 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 101.0"
+  s.dependency "MapboxNavigationNative", "103.0.1"
```

Comment:
Why is it the exact version?

Reply:
Because it's for a pre-release, we should use the exact version.

Reply:
Thanks for the suggestion, I removed the exact version requirement for the `MapboxNavigationNative`.